dataValidator/helpers/dataLoader/readScenario.js

```diff
@@ -15,7 +15,7 @@ const ifDirectoryThenValidateNestedFiles = fullFileName => {
 const readScenario = (fileName, dirName) => {
   const fullFileName = `${dirName}/${fileName}`;
 
-  const ignoreDirs = ['onward-journeys', 'frontpage', 'most'];
+  const ignoreDirs = ['onward-journeys', 'frontpage', 'most', 'mediaassetpage'];
```

Comment:
highlights some non-standard naming of our data directories - should probably be `media-asset-page` to match  `onward-journeys`